scripts/deploy.sh

```diff
@@ -32,13 +32,24 @@ else
   ENV_TARGET="staging"
 fi
 
+# For staging deploys we pass all environment variables which start with STAGING_ and for production
+# deploys we pass all environment variables which start with PROD_. We also remove the prefix so that
+# the deploy script is identical in both cases
+
+if [ "$TARGET" = "staging" ]; then
+  ENV_PREFIX = "STAGING_"
+elif [ "$TARGET" = "prod" ]; then
+  ENV_PREFIX = "PROD_"
+fi
+
+# Extracting environment variables
+DEPLOYMENT_ENV_VARS = `env | grep $ENV_PREFIX | awk '{print "-e ",$1}' ORS=' ' | sed -e "s/$ENV_PREFIX//g"`
+ENV_VARS = "$TRAVIS_ENV_VARS $DEPLOYMENT_ENV_VARS"
+
+
 ## Custom variables passed for netlify
 if [ "$TARGET" = "netlify" ]; then
-  NETLIFY_SITE_ID=$NETLIFY_STAGING_SITE_ID
```

Comment:
Will we no longer need to set `NETLIFY_SITE_ID`?

Reply:
I added `STAGING_NETLIFY_SITE_ID` and `PROD_NETLIFY_SITE_ID` to travis config for consistency!